docs/source/reference/envs.rst

```diff
@@ -195,27 +195,30 @@ in the environment. The keys to be included in this inverse transform are passed
 
     Transform
     TransformedEnv
-    RewardClipping
-    Resize
+    BinarizeReward
```

Comment:
Curious how are these ordered now ?

Reply:
First the two classes that you must know about, then the rest in alphabetic order